deeplearning4j-scaleout/spark/dl4j-spark-ml/src/main/spark-1/org/deeplearning4j/spark/ml/impl/SparkDl4jNetwork.scala

```diff
@@ -0,0 +1,32 @@
+package org.deeplearning4j.spark.ml.impl
+
+import org.apache.spark.ml.param.ParamMap
+import org.apache.spark.ml.util._
+import org.apache.spark.mllib.linalg.Vector
+import org.apache.spark.mllib.regression.LabeledPoint
+import org.apache.spark.sql.{DataFrame, Row}
+import org.deeplearning4j.spark.impl.multilayer.SparkDl4jMultiLayer
+import org.deeplearning4j.spark.ml.utils.DatasetFacade
+
+
+final class SparkDl4jNetwork(override val uid: String) extends SparkDl4jNetworkWrapper[Vector, SparkDl4jNetwork, SparkDl4jModel](uid) {
```

Comment:
For consistency in classes that have to be maintained for 2 Spark versions, elsewhere in the code base we use `MyClassAdapter` for what you name `MyClassWrapper`, and `MyClassAdaptee` for what you name `MyClass`. Would you mind shifting your names to this convention? We find it a useful reminder to keep both implementations in sync.

Also, why the final?